test/e2e/getenvoy_extension_build_test.go

```diff
@@ -34,19 +34,17 @@ func TestGetEnvoyExtensionBuild(t *testing.T) {
 		test := test // pin! see https://github.com/kyoh86/scopelint for why
 
 		t.Run(test.String(), func(t *testing.T) {
+			// TODO: uses Docker, but may be possible to run parallel
 			workDir, removeWorkDir := RequireNewTempDir(t)
 			defer removeWorkDir()
 
-			_, revertChDir := RequireChDir(t, workDir)
```

Comment:
chdir is not safe in parallel, so that's why I had to redo variables everywhere to be explicit.